README.rst

```diff
@@ -0,0 +1,126 @@
+dbsnap-verify
+#####################
+
+verify AWS RDS DB snapshots.
+
+This program may execute as either a CLI tool or an AWS Lambda.
+
+The program uses the "unreliable town clock" pattern.
+This means it expects to be "chimed" every 15m or so using a mechanism like crontab or a scheduled Cloudwatch Rule. 
+
+The program is a state machine, it changes behavior based on the current state.
+It will store execution state as a JSON document in a local file or in ``S3``.
+
+.. contents::
+
+Install
+===============
+
+You may install this tool into your Python environment by running::
+
+ python setup.py develop
+ 
+CLI Tool
+===============
+
+Verify install::
+
+ ./dbsnap-version --help
+
+The tool expects to be passed a JSON config file that mirrors the same config that the AWS Lambda would be provided.
+
+An example `dbsnap-verify-config.json <https://github.com/remind101/dbsnap-verify/blob/import/tests/fixtures/config_or_event.json>`_ may be found here.
+
+
+AWS Lambda
+===============
+
+You may build an AWS Lambda zip. This command assumes you have Python ``virtualenv`` installed::
+
+ make build-lambda
+
+In our case use use a Cloudwatch Rule Event Trigger to invoke our Lambda at a ``rate(15 minutes)``.
+
+The payload of this Cloudwatch Rule is a Static JSON value and is in the same form as the config used for the CLI.
+
+An example `dbsnap-verify-config.json <https://github.com/remind101/dbsnap-verify/blob/import/tests/fixtures/config_or_event.json>`_ may be found here.
+
+config.json
```

Comment:
The CLI and the AWS Lambda both use a JSON document for configuration. This allows us to have a single AWS Lambda function. The function accepts a JSON playload from cloudwatch where the config is stored.

We capture this JSON document in "config management" (stacker). If you look at https://github.com/remind101/stacks/pull/1027/files#diff-315f604a5ab6b943fe1b9856b0976b19R101

Reply:
After explaining why the JSON is used (in both the CLI and Lambda) because that is what cloudwatch events use, Mike agreed this was a better approach.